Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "cloudflare"
-version = "0.3.1"
+version = "0.4.1"
```

Comment:
Isn't this something we only bump up during release?

Reply:
Good point